docs/html/cli/pip_search.rst

```diff
@@ -3,23 +3,15 @@
 ==========
 pip search
 ==========
-
-
-
-Usage
-=====
-
-.. tab:: Unix/macOS
-
-   .. pip-command-usage:: search "python -m pip"
-
-.. tab:: Windows
-
-   .. pip-command-usage:: search "py -m pip"
+.. deprecated:: 20.3.4
 
 
 Description
 ===========
+.. warning::
```

Comment:
```suggestion
===========

.. attention::
```

Let's give the heading some space, and change this from a warning to something more appropriate. :)